File: web-sources/s8-web-front/carbide/cube/CubeElement.js

import { S8WebFront } from '/s8-web-front/S8WebFront.js';
import { NeObject } from '/s8-io-bohr-neon/NeObject.js';



/**
 * 
 */
S8WebFront.CSS_import('/s8-web-front/carbide/cube/Cube.css');

export class CubeElement extends NeObject {


    constructor(){
        super();
    }

    S8_render(){ /* continuous rendering approach... */ }

    S8_set_zIndex(level){
        this.getEnvelope().style.zIndex = level;
    }

    S8_dispose(){ /* continuous rendering approach... */ }
}

File: mod-sources/com.s8.web.front/com/s8/web/front/carbide/cube/CubeElement.java
package com.s8.web.front.carbide.cube;

import com.s8.api.objects.web.WebS8Object;
import com.s8.api.objects.web.WebS8Session;

public class CubeElement extends WebS8Object {

	
	public CubeElement(WebS8Session branch, String typeName) {
		super(branch, typeName);
	}

	
	public void setZIndex(int zIndex) {
		vertex.fields().setUInt8Field("zIndex", zIndex);
	}


}
